prow/cmd/deck/static/pr/pr.ts

```diff
@@ -867,10 +867,10 @@ function createQueriesTable(prLabels: { Label: Label }[], query: ProcessedQuery)
   const col3 = document.createElement("td");
 
   const body = document.createElement("tbody");
-    const row = document.createElement("tr");
-    row.appendChild(createMergeLabelCell(query.labels, true));
-    row.appendChild(createMergeLabelCell(query.missingLabels));
-    body.appendChild(row);
+  const row = document.createElement("tr");
+  row.appendChild(createMergeLabelCell(query.labels, true));
+  row.appendChild(createMergeLabelCell(query.missingLabels));
+  body.appendChild(row);
```

Comment:
Why those changes shown up?

Reply:
These changes were added at executed:

```bash
make -C releng prepare-release-branch
```

following the [release-branch-jobs guide](https://github.com/kubernetes/test-infra/blob/master/releng/README.md#release-branch-jobs) .